<!DOCTYPE html>
<html>
<head><meta name="generator" content="Hexo 3.9.0">
  <meta charset="utf-8">
  

  
  <title>Hexo</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <meta name="description" content="BackgroundChIP-Seq is  Quality ControlNo, ChIPQC does not compute these metrics. It does compute FRiP (fraction of reads in peaks) and duplication rate (fraction of reads that map to the same location">
<meta property="og:type" content="article">
<meta property="og:title" content="Hexo">
<meta property="og:url" content="http://yoursite.com/2019/08/14/ChIP-Seq/index.html">
<meta property="og:site_name" content="Hexo">
<meta property="og:description" content="BackgroundChIP-Seq is  Quality ControlNo, ChIPQC does not compute these metrics. It does compute FRiP (fraction of reads in peaks) and duplication rate (fraction of reads that map to the same location">
<meta property="og:locale" content="en">
<meta property="og:updated_time" content="2019-08-14T09:46:28.000Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Hexo">
<meta name="twitter:description" content="BackgroundChIP-Seq is  Quality ControlNo, ChIPQC does not compute these metrics. It does compute FRiP (fraction of reads in peaks) and duplication rate (fraction of reads that map to the same location">
  
    <link rel="alternate" href="/atom.xml" title="Hexo" type="application/atom+xml">
  
  
    <link rel="icon" href="/favicon.png">
  
  
    <link href="//fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet" type="text/css">
  
  <link rel="stylesheet" href="/css/style.css">
</head>
</html>
<body>
  <div id="container">
    <div id="wrap">
      <header id="header">
  <div id="banner"></div>
  <div id="header-outer" class="outer">
    <div id="header-title" class="inner">
      <h1 id="logo-wrap">
        <a href="/" id="logo">Hexo</a>
      </h1>
      
    </div>
    <div id="header-inner" class="inner">
      <nav id="main-nav">
        <a id="main-nav-toggle" class="nav-icon"></a>
        
          <a class="main-nav-link" href="/">Home</a>
        
          <a class="main-nav-link" href="/archives">Archives</a>
        
      </nav>
      <nav id="sub-nav">
        
          <a id="nav-rss-link" class="nav-icon" href="/atom.xml" title="RSS Feed"></a>
        
        <a id="nav-search-btn" class="nav-icon" title="Search"></a>
      </nav>
      <div id="search-form-wrap">
        <form action="//google.com/search" method="get" accept-charset="UTF-8" class="search-form"><input type="search" name="q" class="search-form-input" placeholder="Search"><button type="submit" class="search-form-submit">&#xF002;</button><input type="hidden" name="sitesearch" value="http://yoursite.com"></form>
      </div>
    </div>
  </div>
</header>
      <div class="outer">
        <section id="main"><article id="post-ChIP-Seq" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2019/08/14/ChIP-Seq/" class="article-date">
  <time datetime="2019-08-14T09:46:28.000Z" itemprop="datePublished">2019-08-14</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
    <div class="article-entry" itemprop="articleBody">
      
        <h2 id="Background"><a href="#Background" class="headerlink" title="Background"></a>Background</h2><p>ChIP-Seq is </p>
<h2 id="Quality-Control"><a href="#Quality-Control" class="headerlink" title="Quality Control"></a>Quality Control</h2><p>No, <code>ChIPQC</code> does not compute these metrics. It does compute FRiP (fraction of reads in peaks) and duplication rate (fraction of reads that map to the same location as another read)..</p>
<h4 id="Mapping-Qualit："><a href="#Mapping-Qualit：" class="headerlink" title="Mapping Qualit："></a>Mapping Qualit：</h4><p>Uniquely mappable reads number / rate,  overall map rate </p>
<h4 id="Sequence-Quality"><a href="#Sequence-Quality" class="headerlink" title="Sequence Quality"></a>Sequence Quality</h4><p>base quality and GC content</p>
<h4 id="Library-Complexity-PCR-bottleneck-cofficient-PBC"><a href="#Library-Complexity-PCR-bottleneck-cofficient-PBC" class="headerlink" title="Library Complexity: PCR bottleneck cofficient (PBC)"></a>Library Complexity: PCR bottleneck cofficient (PBC)</h4><p>A measure of library complexity, i.e. how skewed the distribution of read counts per location is towards 1 read per location.<br>$$<br>PBC = N_1/N_d<br>$$<br>(where $N_1$ = number of genomic locations to which EXACTLY one unique mapping read maps, and Nd = the number of genomic locations to which AT LEAST one unique mapping read maps, i.e. the number of non-redundant, unique mapping reads).</p>
<p>Provisionally,</p>
<ul>
<li>0-0.5 is severe bottlenecking,</li>
<li>0.5-0.8 is moderate bottlenecking, </li>
<li>0.8-0.9 is mild bottlenecking, </li>
<li>0.9-1.0 is no bottlenecking.</li>
</ul>
<p>Very low values can indicate a technical problem, such as PCR bias, or a biological finding, such as a very rare genomic feature. Nuclease-based assays (DNase, MNase) detecting features with base-pair resolution (transcription factor footprints, positioned nucleosomes) are expected to recover the same read multiple times, resulting in a lower PBC score for these assays. Note that the most complex library, random DNA, would approach 1.0, thus the very highest values can indicate technical problems with libraries. It is the practice for some labs outside of ENCODE to remove redundant reads; after this has been done, the value for this metric is 1.0, and this metric is not meaningful. 82% of TF ChIP, 89% of His ChIP, 77% of DNase, 98% of FAIRE, and 97% of control ENCODE datasets have no or mild bottlenecking.</p>
<h4 id="ChIP-enrichment"><a href="#ChIP-enrichment" class="headerlink" title="ChIP enrichment"></a>ChIP enrichment</h4><p>Sufficient number of peaks ( &gt;500 ) with good enrichment (like 10 fold change )</p>
<h4 id="Signal-to-Noise-Ratio"><a href="#Signal-to-Noise-Ratio" class="headerlink" title="Signal to Noise Ratio"></a>Signal to Noise Ratio</h4><p>Fraction of reads in peaks (FRiP)</p>
<h4 id="Regulatory-Region"><a href="#Regulatory-Region" class="headerlink" title="Regulatory Region:"></a>Regulatory Region:</h4><p>‘DNase-seq’ union hypersensitive sites’ (DHS) overlapped ration in top 5000 peaks</p>
<h4 id="Normalized-Strand-Cross-correlation-coefficient-NSC"><a href="#Normalized-Strand-Cross-correlation-coefficient-NSC" class="headerlink" title="Normalized Strand Cross-correlation coefficient (NSC):"></a>Normalized Strand Cross-correlation coefficient (NSC):</h4><p>A measure of enrichment derived without dependence on prior determination of enriched regions. Forward and reverse strand read coverage signal tracks are computed (number of unique mapping read starts at each base in the genome on the + and - strand counted separately). The forward and reverse tracks are shifted towards and away from each other by incremental distances and for each shift, the Pearson correlation coefficient is computed. In this way, a cross-correlation profile is computed, representing the correlation between forward and reverse strand coverage at different shifts. The highest cross-correlation value is obtained at a strand shift equal to the predominant fragment length in the dataset as a result of clustering/enrichment of relative fixed-size fragments around the binding sites of the target factor or feature.</p>
<p>The NSC is the ratio of the maximal cross-correlation value (which occurs at strand shift equal to fragment length) divided by the background cross-correlation (minimum cross-correlation value over all possible strand shifts). Higher values indicate more enrichment, values less than 1.1 are relatively low NSC scores, and the minimum possible value is 1 (no enrichment). This score is sensitive to technical effects; for example, high-quality antibodies such as H3K4me3 and CTCF score well for all cell types and ENCODE production groups, and variation in enrichment in particular IPs is detected as stochastic variation. This score is also sensitive to biological effects; narrow marks score higher than broad marks (H3K4me3 vs H3K36me3, H3K27me3) for all cell types and ENCODE production groups, and features present in some individual cells, but not others, in a population are expected to have lower scores.</p>
<h4 id="Relative-Strand-Cross-correlation-coefficient-RSC"><a href="#Relative-Strand-Cross-correlation-coefficient-RSC" class="headerlink" title="Relative Strand Cross-correlation coefficient (RSC):"></a>Relative Strand Cross-correlation coefficient (RSC):</h4><p>A measure of enrichment derived without dependence on prior determination of enriched regions. Forward and reverse strand read coverage signal tracks are computed (number of unique mapping read starts at each base in the genome on the + and - strand counted separately). The forward and reverse tracks are shifted towards and away from each other by incremental distances and for each shift, the Pearson correlation coefficient is computed. In this way, a cross-correlation profile is computed representing the correlation values between forward and reverse strand coverage at different shifts. The highest cross-correlation value is obtained at a strand shift equal to the predominant fragment length in the dataset as a result of clustering/enrichment of relative fixed-size fragments around the binding sites of the target factor. For short-read datasets (&lt; 100 bp reads) and large genomes with a significant number of non-uniquely mappable positions (e.g., human and mouse), a cross-correlation phantom-peak is also observed at a strand-shift equal to the read length. This read-length peak is an effect of the variable and dispersed mappability of positions across the genome. For a significantly enriched dataset, the fragment length cross-correlation peak (representing clustering of fragments around target sites) should be larger than the mappability-based read-length peak.</p>
<p>The RSC is the ratio of the fragment-length cross-correlation value minus the background cross-correlation value, divided by the phantom-peak cross-correlation value minus the background cross-correlation value. The minimum possible value is 0 (no signal), highly enriched experiments have values greater than 1, and values much less than 1 may indicate low quality.</p>
<h2 id="trick"><a href="#trick" class="headerlink" title="trick"></a>trick</h2>
      
    </div>
    <footer class="article-footer">
      <a data-url="http://yoursite.com/2019/08/14/ChIP-Seq/" data-id="cjzccoo18000278kbh5b69nm1" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
    
<nav id="article-nav">
  
  
    <a href="/2019/08/14/Blog-test/" id="article-nav-older" class="article-nav-link-wrap">
      <strong class="article-nav-caption">Older</strong>
      <div class="article-nav-title">Blog test</div>
    </a>
  
</nav>

  
</article>

</section>
        
          <aside id="sidebar">
  
    

  
    

  
    
  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Archives</h3>
    <div class="widget">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">August 2019</a></li></ul>
    </div>
  </div>


  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Recent Posts</h3>
    <div class="widget">
      <ul>
        
          <li>
            <a href="/2019/08/14/ChIP-Seq/">(no title)</a>
          </li>
        
          <li>
            <a href="/2019/08/14/Blog-test/">Blog test</a>
          </li>
        
          <li>
            <a href="/2019/08/14/hello-world/">Hello World</a>
          </li>
        
      </ul>
    </div>
  </div>

  
</aside>
        
      </div>
      <footer id="footer">
  
  <div class="outer">
    <div id="footer-info" class="inner">
      &copy; 2019 John Doe<br>
      Powered by <a href="http://hexo.io/" target="_blank">Hexo</a>
    </div>
  </div>
</footer>
    </div>
    <nav id="mobile-nav">
  
    <a href="/" class="mobile-nav-link">Home</a>
  
    <a href="/archives" class="mobile-nav-link">Archives</a>
  
</nav>
    

<script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>


  <link rel="stylesheet" href="/fancybox/jquery.fancybox.css">
  <script src="/fancybox/jquery.fancybox.pack.js"></script>


<script src="/js/script.js"></script>



  </div>
</body>
</html>